loops-and-arrays.js

```diff
@@ -53,3 +57,11 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 24 is 13824',
 //   'Het volume van 11 is 1331'
 // ]
+
+const squares = [30, 2, 8, 24, 11];
+
+console.log(squares);
```

Comment:
het is zonder dit hier te loggen. anders krijg je onnodig values in je terminal. beter om af te halen waneer je een keer heb gecheckt hebt.